src/tpm2_wrap.c

```diff
@@ -415,6 +430,24 @@ int wolfTPM2_SetAuthHandle(WOLFTPM2_DEV* dev, int index,
     return wolfTPM2_SetAuth(dev, index, TPM_RS_PW, auth, 0, name);
 }
 
+int wolfTPM2_SetNameHandle(WOLFTPM2_DEV* dev, int index,
```

Comment:
Please rename this to `wolfTPM2_SetAuthHandleName`. It is just setting the name for an existing auth handle.

Reply:
OK. I think this name fits better, however the justification for me is different - We use the AUTH structure to pass the NAME of a TPM Handle down to our parameter encryption implementation.  Typically, the NAME of a TPM Handle is not part of a AUTH structure. In our case, it is a good solution to fetch necessary info for our parameter encryption.